include/phasar/PhasarLLVM/DataFlowSolver/Mono/Solver/InterMonoSolver.h

```diff
@@ -344,16 +346,67 @@ template <typename AnalysisDomainTy, unsigned K> class InterMonoSolver {
     }
   }
 
-  mono_container_t
-  summarize(/*Function CalleeTarget, mono_container_t FactAtCall */) {
-    // spawn a new analysis with its own worklist
-  }
+   mono_container_t summarize(f_t CalleeTarget, mono_container_t FactAtCall) {
+    mono_container_t ResultSummary;
+   // auto EntryPoint = ICF->getFunctionName(CalleeTarget); //initialize the entrypoint to the name of CalleeTarget function
```

Comment:
You first of all still need to map facts to the callee. Then, you can run an analysis of the callee. You also need to specify which function the IntraMonoSolver should process.